content/minfraud/minfraud-legacy.mdx

```diff
@@ -322,8 +322,8 @@ Here is an example of the output with just a few fields:
 
 `riskScore=13.2;distance=6;countryMatch=Yes;countryCode=US;freeMail=Yes`
 
-The API version column indicates in what version of the API a field was added.
-You can explicitly
+The API version column specifies the earliest API version in which a field is
+available. You can explicitly
```

Comment:
This isn't your fault, but this is a bit confusing for things like `explanation` that have since been removed. @christophermluna, do you think we should just ignore this issue? `explanation` and `score` were removed in 1.3. This is mentioned in the description, but it isn't exactly clear when they were available.

Reply:
I'll go ahead and merge this as the issue is somewhat orthogonal and can be fixed later.